signup.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My component library</title>
+    <script src="https://kit.fontawesome.com/917a992f26.js" crossorigin="anonymous"></script>
+    <link href="./styles.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;900&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="container">
+    <h3> Sign Up</h3>
+    <h5>Already have an account?<a href="login.html"> Login</a> instead</h5>
+    <form action="home.html" method="post">
+        <div>
+            <input type="text" id="first_name" name="first_name" required>
```

Comment:
why different naming convention for id and class. In class you are using hyphens - and in id underscore. Use one for consistency, correct me if there's a reason behind it